docs/development.md

```diff
@@ -55,10 +54,42 @@ The box can now be accessed via ssh and the Rails server started directly (this
 ```sh
 vagrant ssh centos7-devel
 cd foreman
-bundle exec foreman start
 ```
 
-The Foreman UI is then accessible on `https://centos7-devel.<hostname>.example.com/`, where `<hostname>` is the shortname of your workstation.  While using the webpack dev server, you must use HTTPS and the hostname, as well as visiting `https://centos7-devel.<hostname>.example.com:3808` to accept the self-signed certificate.
+### Starting the dev server
+
+Our backend requires a rails server to be running. We use webpack for the more "modern" parts of our frontend UI. 
+[Webpack](https://webpack.js.org/) is a way of bundling up multiple front-end files and assets to a small amount of files to send to the browser. 
+The files that webpack handles are located in `webpack/` directory found in Foreman, Katello,
+and plugin root directories. If you are editing any files in `webpack/`, and want to have your changes refresh automatically, you will need a webpack server running.
+
+Because we are using the webpack server in conjunction with a rails server, there are different ways of starting a server depending on your needs and preferences. The following are the
+instructions for starting the server using a base `centos7-devel` box as a starting point.
+
+#### Run a rails and webpack server together using `foreman start`
+- run `foreman start` in `~/foreman`
+- navigate to `https://centos7-devel.<hostname>.example.com/` (or use your prefered way of navigating your dev server).
```

Comment:
Can we still say "where `<hostname>` is the shortname of your workstation"? It might not be obvious what `<hostname>` means.

Also "preferred way of navigating" is a little confusing, with the `--public centos7-devel.<hostname>.example.com` in our default config, this is the *only* way it will work unless we make `--disable-host-check` the default.

Reply:
Also note that the domain can be overridden. Either at the box level or global settings.

Reply:
>Can we still say "where <hostname> is the shortname of your workstation"? It might not be obvious what <hostname> means.

Sure thing

>Also "preferred way of navigating" is a little confusing, with the --public centos7-devel.<hostname>.example.com in our default config, this is the only way it will work unless we make --disable-host-check the default.

many of us use reverse proxies, which does work with these steps, but you navigate to \<myhypervisor\>:\<port\>, I was trying to support that workflow but maybe its not worth mentioning

Reply:
> many of us use reverse proxies, which does work with these steps, but you navigate to <myhypervisor>:<port>, I was trying to support that workflow but maybe its not worth mentioning

It probably is, since you're not the only one who uses it, but maybe could go in a separate section., and maybe as a separate PR? Since we need to forward 3808 too in 
https://github.com/theforeman/forklift/blob/master/vagrant/lib/forklift/box_distributor.rb#L244, and change the --public entry, maybe with an installer flag.